flow-typed/box-ui-elements.js

```diff
@@ -542,6 +542,32 @@ type Translations = {
     onTranslate?: Function
 };
 
+type OpenWithIntegrations = {
+    items: Array<OpenWithIntegrationItem> 
+}
+
+type OpenWithIntegrationItem = {
+    app_integration: AppIntegrationMiniItem,
+    disabled_reasons: Array<String>,
+    display_order: number,
+    icon: string,
+    is_disabled: boolean,
+    should_show_consent_popup: boolean
+}
+
+type AppIntegrationItem = {
+    type: 'app_integration',
+    id: string,
+    title?: string,
+    description?: string
+}
+
+type AppIntegrationMiniItem = {
+    type: 'app_integration',
+    // Known issue, should be a string
+    id: number
```

Comment:
Can we make this `number | string` for forwards compatibility?

Reply:
yes, that is possible but then we won't see errors when the fix is in, so we might forget to move it :)